GOVERNANCE.md

```diff
@@ -170,20 +183,23 @@ yourself from the MAINTAINERS file.
 Similar to the procedure for adding new maintainers, existing maintainers can
 be removed from the list if they do not show significant activity on the
 project. Periodically, the maintainers review the list of maintainers and their
-activity over the last three months.
+activity over the last year.
 
 If a maintainer has shown insufficient activity over this period, a neutral
 person will contact the maintainer to ask if they want to continue being
 a maintainer. If the maintainer decides to step down as a maintainer, they
 open a pull request to be removed from the MAINTAINERS file.
 
-If the maintainer wants to remain a maintainer, but is unable to perform the
-required duties they can be removed with a vote of at least 66% of
-the current maintainers. An e-mail is sent to the
-mailing list, inviting maintainers of the project to vote. The voting period is
-five business days. Issues related to a maintainer's performance should be
-discussed with them among the other maintainers so that they are not surprised
-by a pull request removing them.
+If a committer is unable to continue participating in project level votes but
+wishes to remain a maintainer on the project, they may convert their role to
+a reviewer until they are able to perform committer duties again.
+
+If an inactive maintainer wants to remain in their current role or does not open
+their own pull request to change roles, another maintainer may open up a pull
+request to the MAINTAINERS file to change their role. Role changes and removals
+must be approved by 2/3 of the current committers. The voting and discussion
+period is five business days, after which a committer will verify the votes,
```

Comment:
"business days" differ across countries.  Just having "seven days" might be more clear?

Reply:
agree, I'd like to see this a week or seven days.. just in case someone we need to vote is on vacation from monday through friday..